src/osuTK.iOS/iOSGameView.cs

```diff
@@ -975,7 +988,8 @@ protected virtual void OnUnload(EventArgs e)
         public event EventHandler<MouseButtonEventArgs> MouseUp;
         public event EventHandler<MouseMoveEventArgs> MouseMove;
         public event EventHandler<MouseWheelEventArgs> MouseWheel;
-        public event EventHandler<FileDropEventArgs> FileDrop;
+        public event EventHandler<FileDropEventArgs> FileDrop = delegate { };
```

Comment:
Why?

Reply:
I assumed from `NativeWindow.cs` the same approach would've been required. Upon further testing without this. It seems like this change was not needed. A push will follow reverting this change.